modules/core/src/stat.cpp

```diff
@@ -757,38 +757,37 @@ struct SumSqr_SIMD<uchar, int, int>
 
         int x = 0;
         __m128i v_zero = _mm_setzero_si128(), v_sum = v_zero, v_sqsum = v_zero;
+        const int len_16 = len & 0xfffffff0;
```

Comment:
this construction implies that len is 32-bit integer; I would use more safe `len & ~15`